ReactiveCocoa/Swift/Property.swift

```diff
@@ -25,11 +25,19 @@ public protocol PropertyProtocol: class {
 }
 
 /// Represents an observable property that can be mutated directly.
-public protocol MutablePropertyProtocol: PropertyProtocol {
+public protocol MutablePropertyProtocol: BindingConsumer, PropertyProtocol {
 	/// The current value of the property.
 	var value: Value { get set }
 }
 
+extension MutablePropertyProtocol {
```

Comment:
I would declare the conformance to `BindingConsumer` here since this is where it's implemented.